### PR TITLE
Promoting class fields to Stage 3

### PR DESCRIPTION
**Pull request by afmenez**
The "class fields" proposal was promoted to Stage 3 last week.  I believe none of the other changes affect this table because there are no tests for them yet (see #1034).

### CODE REVIEW

**Review by chicoxyzzy — APPROVED** [review]

**Review by ljharb — CHANGES_REQUESTED** [review]
The shorthand syntax was deferred to a separate proposal, so that test needs to be deleted.

**Comment by afmenez** [comment]
Sorry, can you elaborate?  Do you mean one of the subtests must be split to a separate proposal? Which subtest (I assume it is the last one, "private fields initializers")?  Which separate proposal?

**Comment by chicoxyzzy** [comment]
Does new proposal for shorthand syntax of class fields already exist?

**Comment by ljharb** [comment]
The shorthand proposal does not yet exist; for now we should delete or fix any test that does not have `this.` in front of the #.

**Comment by afmenez** [comment]
So, the only change would be in the last test (private fields initializers)
```
class C {
-   #x = 42;
+   this.#x = 42;
    x() {
        return this.#x;
    }
}
return new C().x() === 42;
```
Is that it?

**Comment by ljharb** [comment]
Yep!

It'd also be good (but not necessary) to also add a test that accesses a private field on another instance of the same class.

**Comment by afmenez** [comment]
I did fix the shorthand use, but I left the additional tests to a separate patch, since it is not the intention of this one.

File: data-esnext.js [hunk 1]
```diff
@@ -1902,7 +1902,7 @@ exports.tests = [
       name: 'private fields initializers',
       exec: function () {/*
         class C {
-          #x = 42;
+          this.#x = 42;
```
Comment (ljharb) [inline data-esnext.js]: oh crap sorry, this is my bad :-( the shorthand is `#x` in code - but in the class body, `#x = 42` is already correct (－‸ლ) apologies

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by ljharb** [comment]
k, i excised the extra commit, sorry about that

**Comment by afmenez** [comment]
No problem.

**Pull request merged**